backend/test/integration/defenceController.test.ts

```diff
@@ -0,0 +1,312 @@
+		level: LEVEL_NAMES;
+		chatHistory: ChatMessage[];
+		defences?: Defence[];
+		sentEmails: EmailInfo[];
+	}
+}
+
+function responseMock() {
+	return {
+		send: jest.fn(),
+		status: jest.fn().mockReturnThis(),
+	} as unknown as Response;
+}
+
+describe('The correct levels can have their defences changed', () => {
+	[LEVEL_NAMES.LEVEL_1, LEVEL_NAMES.LEVEL_2].forEach((level) => {
+		test(`GIVEN level ${
+			level + 1
+		} WHEN attempt to activate a defence THEN defence is not activated`, () => {
+			const req = {
+				body: {
+					defenceId: DEFENCE_ID.CHARACTER_LIMIT,
+					level,
+				},
+				session: {
+					levelState: getInitialLevelStates(),
+				},
+			} as unknown as DefenceActivateRequest;
+
+			const res = responseMock();
+
+			handleDefenceActivation(req, res);
+
+			expect(res.status).toHaveBeenCalledWith(400);
+			expect(res.send).toHaveBeenCalledWith(
+				'You cannot activate defences on this level, because it uses the default defences'
+			);
+		});
+
+		test(`GIVEN level ${
+			level + 1
+		} WHEN attempt to deactivate a defence THEN defence is not activated`, () => {
+			const req = {
+				body: {
+					defenceId: DEFENCE_ID.CHARACTER_LIMIT,
+					level,
+				},
+				session: {
+					levelState: getInitialLevelStates(),
+				},
+			} as unknown as DefenceActivateRequest;
+
+			const res = responseMock();
+
+			handleDefenceDeactivation(req, res);
+
+			expect(res.status).toHaveBeenCalledWith(400);
+			expect(res.send).toHaveBeenCalledWith(
+				'You cannot deactivate defences on this level, because it uses the default defences'
+			);
+		});
+
+		test(`GIVEN level ${
+			level + 1
+		} WHEN attempt to configure a defence THEN defence is not configured`, () => {
+			const req = {
+				body: {
+					defenceId: DEFENCE_ID.CHARACTER_LIMIT,
+					level,
+					config: [],
+				},
+				session: {
+					levelState: getInitialLevelStates(),
+				},
+			} as unknown as DefenceConfigureRequest;
+
+			const res = responseMock();
+
+			handleConfigureDefence(req, res);
+
+			expect(res.status).toHaveBeenCalledWith(400);
+			expect(res.send).toHaveBeenCalledWith(
+				'You cannot configure defences on this level, because it uses the default defences'
+			);
+		});
+
+		test(`GIVEN level ${
+			level + 1
+		} WHEN attempt to reset a defence config item THEN defence config item is not reset`, () => {
+			const req = {
+				body: {
+					defenceId: DEFENCE_ID.CHARACTER_LIMIT,
+					level,
+					configItemId: 'MAX_MESSAGE_LENGTH',
+				},
+				session: {
+					levelState: getInitialLevelStates(),
+				},
+			} as DefenceConfigItemResetRequest;
+
+			const res = responseMock();
+
+			handleResetDefenceConfigItem(req, res);
+
+			expect(res.status).toHaveBeenCalledWith(400);
+			expect(res.send).toHaveBeenCalledWith(
+				'You cannot reset defence config items on this level, because it uses the default defences'
+			);
+		});
+
+		test(`GIVEN level ${
+			level + 1
+		} WHEN attempt to get defence status THEN shoult return undefined`, () => {
```

Comment:
Typo: _should_

Is this logic correct though? Do we not expect a BadRequest response?

Reply:
Irrelevant now, the get defences endpoint no longer exists